src/Microsoft.DotNet.Build.Tasks/Targets/coreAssembly.rsp

```diff
@@ -0,0 +1,2 @@
+﻿# CSC requires the runtimemetadataversion flag when building a core assembly (containing System.Object)
+/runtimemetadataversion:v4.0.30319
```

Comment:
So far we have either MSBuildToolsVersion or VisualStudioVersion.  I've found that both seem to be specified wether building in a vs env or not, so they seem like reasonable approximations.  I'll go with MSBuildToolsVersion for now unless @jasonmalinowski has a better recommendation.